src/templates/prototypes/_buttons.pug

```diff
@@ -45,14 +55,15 @@ block content
         +form('javascript:console.log("search submitted")', 'post')
           .input-group
             +input('search', 'Enter a keyword', 'q', 'search_box')(maxlength='1000', pattern='^(?!.*<script|<Script).*$')
-            //- button [value, style, type, datatype]
-            +button('Search', 'btn--primary', 'submit', 'search')
+            button(type='search', class='btn--white', data-tracking='search')
+              include ../../icons/download.svg
```

Comment:
why 'download.svg' for search components?

Reply:
If the download arrow icon works for both the search and download buttons then that's fine with me (I suppose we could rename it to something more suitable).

Reply:
Wrong one here, forgot to update once we got the icon.